tflint/errors.go

```diff
@@ -52,6 +56,14 @@ func (e *Error) Error() string {
 	return e.Message
 }
 
+type ConfigParseError struct {
+	Detail *hcl.Diagnostics
```

Comment:
Go slices (and maps) are implemented with pointers, sort of like this:

```go
type StringSlice struct {
  items []*string
}
```

You can point a slice to a new list of items, but when you pass it around you're not copying the items themselves, just pointers. This is why you'll often see `[]*Widgets` in Go code but rarely `*[]Widgets`.

I'll take care of converting this, just wanted to leave a note for myself and an explanation if you're interested.

Reply:
Ahh, I missed that `hcl.Diagnostics` is just a typedef. Fixed!